client/app/index.html

```diff
@@ -221,6 +222,8 @@ <h3 translate>About</h3>
             'liveblog.posts',
             'liveblog.blog',
             'liveblog.themes',
+            'liveblog.freetypes',
```

Comment:
@nistormihai whatever you say, you can't deny that having two modules, one called `liveblog.freetypes` and the other one called `liveblog.freetype` is a source of confusion and will definetly cause trouble in the foreseeable future.